src/test/java/com/google/javascript/cl2dts/types.js

```diff
@@ -59,3 +59,13 @@ types.j = {a: 'a'};
  * @template K,V
  */
 types.objectWithGenericKeyType = function(obj) { };
+
+/**
+ * @const
```

Comment:
Maybe add a comment explaining that, or at least documenting that this was necessary?